File: POO/U1/Practica/EJ3/Categoria.py
class Categoria:
    def __init_(self, titulo):
        self.__titulo = titulo
        self.__series = []
        
    def agregar_serie(self,serie):
        self.__series.append(serie)

    def imprimir(self):
        print(self.__titulo)
        for ser in self.__series:
            ser.imprimir()

File: POO/U1/Practica/Categoria.java
import java.util.ArrayList;

public class Categoria {
    private String titulo;
    private ArrayList<Serie> series;

    public Categoria(String titulo) {
        this.titulo = titulo;
        this.series = new ArrayList<>();
    }

    public void agregarSerie(Serie ser) {
        this.series.add(ser);
    }

    public void imprimir() {

    }
}
